php/class-fieldmanager-richtextarea.php

```diff
@@ -1,9 +1,9 @@
 <?php
 
 /**
- * Use the WordPress Editor in Fieldmanager.
+ * TinyMCE visual editor which submits HTML.
```

Comment:
It may be worth keeping a reference to the fact that we use the WP core editor implementation? Or if we'd rather keep it abstract it may make sense to just call it a WYSIWYG or something.